src/sqlfluff/rules/L059.py

```diff
@@ -28,14 +38,63 @@ class Rule_L059(BaseRule):
 
         SELECT 123 as foo
 
+    When ``force_quote_identifier = True``, the quotes are always necessary, no
+    matter if the identifier is valid, a reserved keyword, or contains special
+    characters.
+
+    | **Anti-pattern**
+    | In this example, a valid unquoted identifier,
+    | that is also not a reserved keyword, is required to be quoted.
+
+    .. code-block:: sql
+
+        SELECT 123 as foo
+
+    | **Best practice**
+    | Use quoted identifiers.
+
+    .. code-block:: sql
+
+        SELECT 123 as "foo" -- For ANSI, ...
+        -- or
+        SELECT 123 as `foo` -- For BigQuery, MySql, ...
+
     """
 
+    config_keywords = [
```

Comment:
@tunetheweb [Linting tests](https://github.com/sqlfluff/sqlfluff/runs/5037642948?check_suite_focus=true) are outputting this error on this line:
```
./src/sqlfluff/rules/L059.py:64:24: BLK100 Black would make changes.
```

I don't understand what it means. 
Could you help me please?

Reply:
Just run black:
```
black src/sqlfluff/rules/L059.py
```

And it'll magically fix your issues.

I would guess it's saying you can fix this on one line, so black is asking why you are splittng it when you don't need it:

```py
config_keywords = ["force_quote_identifier"]
```

Reply:
Yikes! Not sure how you can code like that. You;'re a better developer than me! ;-)

Ran `black` for you there and committed.

Reply:
I'm PHP oriented and my workspace is dedicated to PHP then, not Python :wink: . I usually don't work editing files 1 by 1 :laughing: .

Thanks for running `black` for me!

Hope this will be ok then.